src/lib.rs

```diff
@@ -1000,6 +1000,82 @@ pub unsafe trait FromBytes: FromZeroes {
     where
         Self: Sized;
 
+    /// Interprets the given `bytes` as a `&Self` without copying.
+    ///
+    /// If `bytes.len() != size_of::<T>()` or `bytes` is
+    /// not aligned to `align_of::<T>()`, this returns `None`.
```

Comment:
Can you reflow this and other comments to 80 columns?

Reply:
Done.